### ISSUE TITLE
ERROR: java.lang.NoSuchMethodException: <unbound>=Configuration.getAllowInsecureTokens();

### ISSUE THREAD

**Issue by tarangchikhalia** [oracle/opengrok#3693]
**Describe the bug**
While testing the authentication using the authentication token in a test environment (insecure), I noticed that the getter method - getAllowInsecureTokens() is not defined in the Configuration.java file.

A snippet of my readonly_configuration.xml file.

```
<void property="authenticationTokens">
   <void method="add">
    <string>foobar</string>
   </void>
</void>

<void property="allowInsecureTokens">
   <void method="add">
    <boolean>true</boolean>
   </void>
</void>
```

OpenGrok version: 1.7.3
JDK version: 11.0
OS: Oracle Linux 7.9

**To Reproduce**
Run the index with the above configuration in the readonly_configuration.xml file.

**Comment by vladak** [comment]
The `allowInsecureTokens` is a simple boolean variable, not a list like `authenticationTokens` so the `add` method is not applicable.

Use this instead:
```xml
<void property="allowInsecureTokens">
    <boolean>true</boolean>
</void>
```

**Comment by vladak** [comment]
will be solved eventually by #2329
